h2o-algos/src/main/java/hex/coxph/CoxPH.java

```diff
@@ -579,6 +579,7 @@ public void computeImpl() {
         // The model to be built
         CoxPHModel.CoxPHOutput output = new CoxPHModel.CoxPHOutput(CoxPH.this, dinfo._adaptedFrame, train(), strataMap);
         model = new CoxPHModel(_result, _parms, output);
+        model.setInputParms(_input_parms);
```

Comment:
I wonder why do you need this

Reply:
Because it's not set there in any other place for CoxPH as far I was searching.

Reply:
this is done in ModelBuilder#computeParameters and should also work for cox, this should not be needed

Reply:
Removed